SECURITY.md

```diff
@@ -1,11 +1,142 @@
+
+If an impact can be caused to any other asset managed by Aptos that isn’t on this table but for which the impact is in the Impacts in Scope section below, you are encouraged to submit it for consideration by the project.
+
+## Out of Scope
+
+The following vulnerabilities are excluded from the rewards for this bug bounty program:
+
+- Attacks that the reporter has already exploited themselves, leading to damage.
+- Attacks requiring access to leaked keys/credentials.
+- Internally known issues, duplicate issues, or issues that have already been made public; in such cases, proof of prior disclosure will be provided.
+- Attacks that rely on social engineering or require physical access to the victim’s device.
+- Information disclosure with minimal security impact (Ex: stack traces, path disclosure, directory listing, logs)
+- Tab-nabbing
+- Vulnerabilities related to auto-fill web forms
+- Vulnerabilities only exploitable on out-of-date browsers or platforms
+- Attacks requiring physical access to the victim device
+- Incorrect data supplied by third party oracles
+
+**********************Blockchain:**********************
+
+- Basic economic governance attacks (e.g., 51% attack).
+- Best practice critiques.
+- Missing or incorrect data in events.
+- Basic economic governance attacks (e.g. 51% attack)
+- Sybil attacks
+- Centralization risk
+
+********************************Smart contracts:********************************
+
+- Incorrect data supplied by third-party oracles (not to exclude oracle manipulation/flash loan attacks; use of such methods to generate critical impacts remain in-scope for this program).
+- Basic economic governance attacks (e.g., 51% attack).
+- Lack of liquidity.
+- Best practice critiques.
+- Missing or incorrect data in events.
+- Incorrect naming (but still correct data) in contracts.
+- Minor rounding errors that don’t lead to substantial loss of funds.
+- Incorrect data supplied by third party oracles
+    - Not to exclude oracle manipulation/flash loan attacks
+
+## Exclusions
+
+The following activities are prohibited by this bug bounty program:
+
+- Any testing with mainnet or public testnet contracts; all testing should be done on [private testnets](https://aptos.dev/nodes/local-testnet/local-testnet-index/)
+- Any testing with pricing oracles or third party smart contracts
+- Attempting phishing or other social engineering attacks against our employees and/or customers
```

Comment:
nit: `Attempting phishing or other social engineering attacks` -> `Attempting to phish or otherwise use social engineering attacks`